.chloggen/ingest_tls_openshift.yaml

```diff
@@ -0,0 +1,27 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use TLS via OpenShift service annotation when gateway/multitenancy is disabled
+
+# One or more tracking issues related to the change
+issues: [963]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  This will use the openshift service annotation to generate TLS certs, in order to use this feature TLS on the 
```

Comment:
I would rephrase this. 

`On OpenShift when operator config `DistributorServingCertsService` is enabled and the following TempoStack CR is used. The operator provisions OpenShift serving certificates for the distributor ingest APIs`

I would as well document what users need to do to configure TLS on their clients.